src/content/developers/tutorials/fetch-price-data-in-solidity-smart-contracts/index.md

```diff
@@ -0,0 +1,65 @@
+
+```
+import "https://github.com/smartcontractkit/chainlink/blob/master/evm-contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
+```
+
+```
+AggregatorV3Interface internal priceFeed;
+```
+
+In the constructor of the <a href="https://docs.chain.link/docs/ethereum-addresses">Price Feed reference contract</a>, we can then initialize the address of the price feed we’re interested in. By exploring the <a href="https://docs.chain.link/docs/ethereum-addresses">Ethereum Price Feeds page</a> in the Chainlink documentation, we can find all the price feed contract addresses that Chainlink currently provides. Navigate to the Kovan portion of the page and choose a price feed. For the purposes of this example, we’ll choose the <a href="https://data.chain.link/eth-usd">ETH/USD</a> Price Feed, whose address is <a href="https://kovan.etherscan.io/address/0x9326BFA02ADD2366b30bacB125260Af641031331">0x9326BFA02ADD2366b30bacB125260Af641031331</a>.
+
+```
+priceFeed = AggregatorV3Interface(0x9326BFA02ADD2366b30bacB125260Af641031331);
+```
+
+A function has been defined called getLatestPrice to obtain the latest price from the Price Feed Aggregator contract that was instantiated in the constructor above. To do this, a new function was defined that calls the <a href="https://docs.chain.link/docs/price-feeds-api-reference#latestrounddata">latestRoundData</a> function from the Aggregator contract. This is the function that returns the current state of the Aggregator contract, and in this case, we are taking the current price and returning it in our consuming function.
+
+```   function getLatestPrice() public view returns (int) {
+        (
+            uint80 roundID, 
+            int price,
+            uint startedAt,
+            uint timeStamp,
+            uint80 answeredInRound
+        ) = priceFeed.latestRoundData();
+        return price;
+    }
+```
+### Deploying and Testing the Smart Contract
+Now we are ready to deploy and test our contract. Compile the contract in Remix, then on the deployment tab, change the environment to “Injected Web3”, and ensure the wallet address below is the one in your MetaMask wallet that contains some testnet ETH obtained earlier, press the deploy button, and follow the steps. The end result is you have your smart contract deployed to the Kovan testnet. You should take note of the deployed contract address via the transaction output in the Remix console. 
+
+Once deployed, we simply need to execute the “getLatestPrice” function. The result should be that the function returns the latest price from the ETH/USD Aggregator contract, which can then be used on-chain in our smart contract. Take note that we didn’t need to send any LINK for the request, and we didn’t even use any ETH either since the transaction is a pure read of the data in the on-chain ETH-USD Aggregator contract.
+
+<p align="center">
+  <img src="./images/eth-usd.png" />
+</p>
```

Comment:
Would just use markdown for this as well
```
![alternate text](src)
```

And by convention we've been keeping images for each tutorial directly in the same folder, ie `/fetch-price-data-in-solidity-smart-contracts/eth-usd.png` if you wouldn't mind relocating the file. 🙏🏻